ftl/common/tar_to_dockerimage.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This package provides DockerImage for examining docker_build outputs."""
+
+import cStringIO
+import json
+import gzip
+
+from containerregistry.client.v2_2 import docker_digest
+from containerregistry.client.v2_2 import docker_image
+from containerregistry.client.v2_2 import docker_http
+from containerregistry.transform.v2_2 import metadata as v2_2_metadata
+
+
+class TarDockerImage(docker_image.DockerImage):
```

Comment:
Maybe rename this "FromFS" or something without tar in the name?

Reply:
done